src/plasma/test/manager_tests.cc

```diff
@@ -217,20 +217,18 @@ TEST read_write_object_chunk_test(void) {
   const char *data = "Hello world!";
   const int data_size = strlen(data) + 1;
   const int metadata_size = 0;
-  plasma_request_buffer remote_buf = {
-      .type = MessageType_PlasmaDataReply,
-      .object_id = oid,
-      .data = (uint8_t *) data,
-      .data_size = data_size,
-      .metadata = (uint8_t *) data + data_size,
-      .metadata_size = metadata_size,
-  };
-  plasma_request_buffer local_buf = {
-      .object_id = oid,
-      .data_size = data_size,
-      .metadata_size = metadata_size,
-      .data = malloc(data_size),
-  };
+  plasma_request_buffer remote_buf;
+  remote_buf.type = MessageType_PlasmaDataReply;
+  remote_buf.object_id = oid;
+  remote_buf.data = (uint8_t *) data;
+  remote_buf.data_size = data_size;
+  remote_buf.metadata = (uint8_t *) data + data_size;
+  remote_buf.metadata_size = metadata_size;
+  plasma_request_buffer local_buf;
+  local_buf.object_id = oid;
+  local_buf.data_size = data_size;
+  local_buf.metadata_size = metadata_size;
+  local_buf.data = (uint8_t *) malloc(data_size);
```

Comment:
might be best to use cpp struct initialization style here (with so many fields).